avocado/plugins/run.py

```diff
@@ -128,10 +128,13 @@ def configure(self, parser):
 
         if multiplexer.MULTIPLEX_CAPABLE:
             mux = parser.add_argument_group('multiplexer use on test execution')
-            mux.add_argument('-m', '--multiplex-files', nargs='*',
+            mux.add_argument('-m', '--multiplex', nargs='*', dest='multiplex_files',
```

Comment:
IMHO, this is purely internal, akin to a variable name. Also, I thought it could be done at `--multiplex-files` removal time, because until then we'll either have to have that or `dest=multiplex` set on `--multiplex-files`.

BTW, we also have a lot of `dest=` in various command line options, and maybe we can add a low hanging fruit card to remove them when not needed or not necessary to have a different name, example:

```
vm.py:        self.vm_parser.add_argument('--vm-port', dest='vm_port',
```